src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
```

Comment:
You don't really need to create a variable for the result. Just return the calculated value 
```
if (...) {
  return ...
}